nginx/sites-enabled/hearts

```diff
@@ -0,0 +1,13 @@
+server {
+    listen 80;
+    server_name lonelyhearts.cards;
```

Comment:
Not sure exactly what this `server_name` is, but will probably have to reconfigure it for whatever our actual domain ends up being.